korge/src/commonMain/kotlin/com/soywiz/korge/tween/tween.kt

```diff
@@ -16,13 +16,13 @@ import kotlin.reflect.*
 class TweenComponent(
 	override val view: View,
 	private val vs: List<V2<*>>,
-	val time: Int? = null,
+	val time: Long? = null,
```

Comment:
Double boxing in JS because of long. Maybe we could use negative numbers and a Double or a TimeSpan to represent the absence of value to totally prevent boxing here.